tendermint/vue
# fix: Inputs UI improvements and selected tokens distance

## Description & Context

The input field for the modal edit did not cover the Figma design, the input field needed to be polished and the behavior improved. It was also necessary to improve the distance between the selected tokens for transfer.

## How To Test

1. Open the Data section;
2. Connect with Keplr;
2. If you don't have items created, create them and click on the three dots icon after to access the menu;
3. Click "Edit" action;
4. The modal window that appears should display the correct design of the inputs;

1. Open the Portfolio section;
2. Connect with Keplr;
3. Choose two tokens to transfer (if you don't have at least two type, you can't test, add another token using cli or other methods to test it correctly);
4. The distance between the two selected tokens must be correct;

## Linked issue

Closes #205, closes #206
